$NetBSD: patch-config___arch-n-opsys,v 1.2 2016/07/03 19:00:25 dholland Exp $

Fix version reporting.
Learn about x86_64.
Support ppc and sparc on NetBSD.

--- config/_arch-n-opsys.dist	2016-07-03 18:45:43.000000000 +0000
+++ config/_arch-n-opsys
@@ -119,13 +119,15 @@ case `uname -s` in
     ;;
   NetBSD)
     case `uname -r` in
-      2*) OPSYS=netbsd2;;
-      3*) OPSYS=netbsd;;
-      *) exit 1;;
+      1.*)  exit 1;;
+      2.*) OPSYS=netbsd2;;
+      *) OPSYS=netbsd;;
     esac
     HEAP_OPSYS=bsd
-    case `uname -m` in
-      *86) ARCH=x86;;
+    case `uname -p` in
+      *86|x86_64) ARCH=x86;;
+      powerpc) ARCH=ppc;;
+      sparc) ARCH=sparc;;
       *) exit 1;;
     esac
     ;;
